### DISCUSSION TITLE
Where to place VAEs?

### DISCUSSION THREAD

**Discussion by jojoha1337** [TheLastBen/fast-stable-diffusion#1050] [Q&A] [answered]
I tried placing a name_of_model.vae.pt in the same folder as the name_of_model.ckpt: MyDrive/models
 
But it didn't appear in the VAE settings window when i launched automatic1111, there were no additional options, just "auto".

I also tried creating a VAE folder, MyDrive/models/VAE, same result.

Is it automatically loaded into the model, or is there something i missed?

**Comment by TheLastBen** [comment]
if you're trying to add the stability new vae for the 1.5, it is automatically imbedded in the default 1.5 model when you download it in the model download cell.

**Reply by jojoha1337** [reply]
Its a different custom VAE.

**Reply by TheLastBen** [reply]
look in the A1111 repo, surely there is a tutorial how to install it

**Reply by jojoha1337** [reply]
Sure, i'll take a look

**Comment by jojoha1337** [comment, marked as answer]
I figured it out and it worked flawlessly.

You have to run everything in the colab as normal, then it will create a "sd" folder in your drive, then you gotta interrupt the last cell that runs the AUTOMATIC1111 stable diffusion ( the Start Stable Diffusion cell to be precise), next step is to simply place your VAE file inside sd/stable-diffusion-webui/models/VAE and run the Start Stable Diffusion cell again, the VAE will now appear in the settings for you to select.

UPDATE
I found out you just have to execute the first (load google drive ) and 2nd cell (Install AUTOMATIC1111 Repo) then the sd folder will appear before you have to execute anything else, then you can just place the VAE inside the sd/stable-diffusion-webui/models/VAE and proceed to run the colab as normal.
